app/assets/stylesheets/components/page_specific/_home_top.scss

```diff
@@ -0,0 +1,184 @@
+// Banner at the top of the home page
+//
+// Styleguide Home Top
+
+.home-top {
+  background: image_url('illustrated_people.png') 0 100% no-repeat;
```

Comment:
Name of function `image_url` should be written in all lowercase letters with hyphens instead of underscores<br>Prefer double-quoted strings